script/QA/QA_14.txt

```diff
@@ -31,77 +32,86 @@ Do you copy, Ciel? Please respond, over.
 [sha:f33e672c6d77f805bef556cd55f96c80cdd2f9ae]{
 -- Page 33, Offset 43378.
 -- 冷静になるンだ。
-You need to calm down.
+You need to calm down.%{n}
 }
 [sha:6945260d138bf92ce316d3232642a967959441b8]{
 -- Page 33, Offset 43379.
 -- 第13回から何も成長していニャいよ？
-Nothing's changed since the 13th lesson, right?
+Did you learn meowthing from the 13th lesson?
 }
 [sha:cdf6364f66395b2550a1b76c38a2c788c3802b7f]{
 -- Page 34, Offset 43380.
 -- え……だってどう見ても
-Huh!? ...But we had such perfect communication
+Huh!? But...
 }
 [sha:a3f3ace02fee45daf507d17de2bf3ccafc760491]{
 -- Page 34, Offset 43381.
 -- ただのパーフェクトコミュニケーションで……
-no matter how you look at it...
+// I think 'perfect communication' here might be 'picked all the right dialog options' imi
+// https://gameo.jp/imassc/178
+// https://lium3-jouhou.com/shinycolors-producer-yositanosiku/
+ It's clearly the right option to max out the%{n}
+affection meter...
 }
 [sha:ad1f861399bffe91922398b63c6dc5f93b90045f]{
 -- Page 35, Offset 43382.
 -- なぜ人類は自分の事だけ棚にあげるのか。
-Why is it that humans only put themselves on a pedestal?
+Why is it that humans are only blind to their%{n}
+own shortcomings?%{n}
 }
 [sha:e2e5a5ee467d38f8d27c8c379f919ee57e2b79ec]{
 -- Page 35, Offset 43383.
 -- 壊れたクレーンかよ。
+// Like a crane game crane? Neco arc high-rise construction services?
+// Or is this connecting on 棚にあげる and now lost in translation?
 What are you, a broken crane?
```

Comment:
I assume that it's just that the idea of breaking a symbol of immortality and longevity.